models/br_denatran_frota/br_denatran_frota__municipio_tipo.sql

```diff
@@ -8,7 +8,7 @@
             "data_type": "int64",
             "range": {
                 "start": 2003,
-                "end": 2024,
+                "end": 2025,
                 "interval": 1,
             },
```

Comment:
Após a primeira materialização em prod, será preciso adicionar esse pre hook         pre_hook="DROP ALL ROW ACCESS POLICIES ON {{ this }}";

Reply:
em ambos os modelos